go/teams/handler.go

```diff
@@ -81,14 +85,15 @@ func HandleSBSRequest(ctx context.Context, g *libkb.GlobalContext, msg keybase1.
 	return nil
 }
 
-func handleSBSSingle(ctx context.Context, g *libkb.GlobalContext, teamID keybase1.TeamID, invitee keybase1.TeamInvitee) error {
+func handleSBSSingle(ctx context.Context, g *libkb.GlobalContext, teamID keybase1.TeamID, invitee keybase1.TeamInvitee) (err error) {
```

Comment:
another way to write this function BTW is to ignore everything in the gregor message other than the invite ID.

Reply:
I find it confusing to have `invitee` and `invite` be one letter off, but have very different meanings and trust assumptions.  Can you rename to `untrustedInviteeFromGregor` and `inviteFromTeamChain`?